articles/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+from users.models import Interest
+
+
+class ArticleStatus(models.Model):
+    label = models.CharField(max_length=100)
+    def __str__(self):
+        return self.label
+
+class Article(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE,related_name='articles_created')
+    status = models.ForeignKey(ArticleStatus,on_delete=models.SET_NULL,blank=True,null=True)
+    created_date = models.DateTimeField(auto_now_add=True)   
+    published_date = models.DateTimeField(blank=True, null=True)
+    title = models.CharField(max_length=100)
+    thumbnail_url = models.URLField(max_length=200)
+    content = models.TextField(max_length=20000,blank=True,null=True)
+    category = models.ForeignKey(Interest,on_delete=models.SET_NULL,blank=True,null=True)
```

Comment:
1. Make this field required.
2. Set the value of on_delete to "models.PROTECT"
3. set the value of  **related_name** to "articles"